client_tools.py

```diff
@@ -104,10 +104,18 @@ def main(args):
                 BotanyImporter(paths=paths, config=pic_config, full_import=full_import,
                                existing_barcodes=existing_barcodes)
             else:
-                date_override = args.date
+                # importing highest csv date
+                try:
+                    # updating tmp jpg folder to avoid clashing with passive importer.
```

Comment:
alternate ./tmp_jpg path used for upload_me and delete_me paths during active imports, to avoid using same folder as passive picturae imports.

Reply:
Use MD5 for directory name so we gaurentee uniqueness. ./tmp_jpg_{md5}